sympy/matrices/expressions/tests/test_matexpr.py

```diff
@@ -360,3 +360,14 @@ def test_issue_2749():
 def test_issue_2750():
     x = MatrixSymbol('x', 1, 1)
     assert (x.T*x).as_explicit()**-1 == Matrix([[x[0, 0]**(-2)]])
+
+
+def test_matrix_equality():
+    from sympy import Eq
```

Comment:
import Eq at top and please refer to issue number: as I noted on the closed PR, if this ever fails it gives an easy reference to the original issue whereas the subject of the test -- matrix equality -- is self evident from the tests added.

Reply:
okay I'll do that.